config/config.go

```diff
@@ -173,8 +173,8 @@ func Load(filename string, loadFromCli ...bool) (*Config, error) {
 	if config.Port <= 0 {
 		config.Port = 19051
 	}
-	if config.SessionTtl <= int64(time.Hour) {
-		config.SessionTtl = int64(time.Hour)
+	if config.SessionTtl <= int64(30 * time.Minute) {
```

Comment:
Maybe we can remove this condition at all now, because session will be kept alive by requests

Reply:
Or, maybe set the default value when config.Instances  <= 0

Reply:
It is ok if i merge it like this for now?

Reply:
> It is ok if i merge it like this for now?

Yeah, go ahead, we can improve this in the feature if we need to